gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,650 @@
+
+== Before You Begin
+
+Before starting the procedures in this guide, ensure that the following prerequisites are in place:
+ 
+* A username and password for your Anypoint Platform organization. If you do not have an Anypoint Platform account, https://anypoint.mulesoft.com/login/signup[create a Trial organization^] on Anypoint Platform.
+//TODO: VERIFY Manage Servers and Read Servers permissions in RTM NO LONGER NEEDED
+
+//os prereqs
+* A Linux or MacOS machine with an x86 processor. M1 and M2 processors are not supported.
+//TODO: VERIFY not Windows x86 -- pending testing 
+
+//docker prereq
+* {empty}
+include::partial$prerequisites.adoc[tag=docker]
++
+Run `docker --version` from a terminal window to find out if this tool is installed. If you need to install Docker, try https://www.docker.com/[Docker Desktop^]. 
+
+//helm prereq
+* {empty}
+include::partial$prerequisites.adoc[tag=helm] 
++
+Run `helm version` from a terminal window to find out if this tool is installed and check the version. To install or upgrade, use the https://helm.sh/[Helm^] website. 
+
+//kubectl prereq to select cluster prior to deployment
+* {empty}
+include::partial$prerequisites.adoc[tag=kubectl]
++
+Run `kubectl version --output=yaml` from a terminal window to find out if this tool is installed. If not, find `kubectl` among the installation tools available through the https://kubernetes.io/docs/tasks/tools/[Kubernetes documentation^]. 
+
+//curl prereq to test the API with flex
+* {empty}
+include::partial$prerequisites.adoc[tag=curl]
++
+Run `curl --version` from a terminal window to find out if this tool is installed. If not, you can download it with a package manager for your machine. 
+
+[[k8-cluster-reqs]]
+== Cluster Requirements
+
+The host for your Kubernetes cluster can be any xref:flex-review-prerequisites.adoc#software-requirements[supported Kubernetes provider], including common cloud-based services, such as Amazon Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE).
+
+To complete the procedures in this guide, you must create a Kubernetes cluster or use an existing one for your Flex Gateway deployment. In this guide, you use Helm to deploy Flex Gateway to a Kubernetes cluster and connect it to Anypoint Platform. 
+
+[[min-cluster-reqs]]
+The following minimum requirements apply to a small Kubernetes cluster used for demonstration purposes in this guide and _do not apply to requirements of a production environment_:
+//TODO: POINT TO PROD REQUIREMENTS ONCE THEY ARE AVAILABLE. 
+//Create story
+
+* At least 1 node.
+
+* Support for the creation of a `Service` resource of the `LoadBalancer` type.
++
+To avoid deployment errors in this guide, your cluster must support this process. However, note that Flex Gateway supports all of the following Ingress Controller services: `ClusterIP`, `NodePort`, `LoadBalancer` (default), `ExternalName`. 
```

Comment:
```suggestion
To avoid deployment errors when using this guide, your cluster must support this process. However, Flex Gateway supports these Ingress Controller services: `ClusterIP`, `NodePort`, `LoadBalancer` (default), `ExternalName`. 
```